Scripts/Haptics/TimedHapticProcess.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace VRTK.Core.Haptics
+{
+    using UnityEngine;
+    using System.Collections;
+
+    /// <summary>
+    /// Processes a given <see cref="HapticProcess"/> repeatidly for a given duration and with a pause interval between each process.
+    /// </summary>
+    public class TimedHapticProcess : HapticProcess
+    {
+        /// <summary>
+        /// The process to utilize.
+        /// </summary>
+        [Tooltip("The process to utilize.")]
+        public HapticProcess hapticProcess;
+        /// <summary>
+        /// The amount of time to keep repeating the process for.
+        /// </summary>
+        [Tooltip("The amount of time to keep repeating the process for."), SerializeField]
+        protected float duration = 1f;
+        /// <summary>
+        /// The amount of time to pause after each process iteration.
+        /// </summary>
+        [Tooltip("The amount of time to pause after each process iteration."), SerializeField]
+        protected float interval = 0.1f;
+
+        /// <summary>
+        /// The amount of time to keep repeating the process for.
+        /// </summary>
+        public float Duration
+        {
+            get { return duration; }
+            set { duration = value; }
+        }
+
+        /// <summary>
+        /// The amount of time to pause after each process iteration.
+        /// </summary>
+        public float Interval
+        {
+            get { return interval; }
+            set { interval = value; }
+        }
+
+        protected Coroutine hapticRoutine;
```

Comment:
Documentation?

Reply:
"A reference to the started routine." perhaps? I mean that's basically all it is, right?

Reply:
Yeah seems fair enough